### DISCUSSION TITLE
server returned error 405

### DISCUSSION THREAD

**Discussion by nnarayanmuchhal** [tulir/whatsmeow#351] [General]
Messages are getting failed with following error...

server returned error 405

What could be possible reason.. and how to resolve...

**Comment by rocymp** [comment]
same problem

**Comment by asorg74** [comment]
same problem.
My app already running for weeks, then suddenly this morning Client Error Client outdated (405) connect failure

**Comment by rafly7** [comment]
same proble here
![Screen Shot 2023-06-12 at 14 04 24](https://github.com/tulir/whatsmeow/assets/39668448/e1f0a151-c807-4682-b078-c171775757b7)

**Comment by hrizal** [comment]
update sources and recompile

Pada tanggal Sen, 12 Jun 2023 pukul 14.04 rafly7 ***@***.***>
menulis:

**Reply by rafly7** [reply]
Thankyou this worked for me

**Comment by suyash2702** [comment]
This worked for me - 
1. go get -u go.mau.fi/whatsmeow@latest
2. resolve compile error in cli.sendMessage
3. compile and run
